src/lib/containers/provenance/ProvenanceGraph.tsx

```diff
@@ -666,16 +378,11 @@ export const ProvenanceGraph = (props: ProvenanceProps) => {
     setEdges,
     initializedPosition,
     reactFlowInstance,
-    findEntityNode,
     rootEntityRefs,
     onNodesChangedListener,
     onEdgesChangedListener,
   ])
 
-  const onInit: OnInit = useCallback(
-    reactFlow => setReactFlowInstance(reactFlow),
-    [],
-  )
   return (
     <SynapseErrorBoundary>
```

Comment:
You could move this ErrorBoundary to the exported wrapper component, which will also capture the potential error thrown by `useGetEntityHeaders`.

Reply:
Done